pkg/reconciler/buildrun/buildrun_suite_test.go

```diff
@@ -1,7 +1,6 @@
 // Copyright The Shipwright Contributors
 //
 // SPDX-License-Identifier: Apache-2.0
-
```

Comment:
We need the empty line between the comment and the package, so that this is not considered a package comment.

Reply:
fixed